core/src/main/scala/com/evolutiongaming/kafka/flow/key/KeyDatabase.scala

```diff
@@ -56,4 +56,11 @@ object KeyDatabase {
 
     }
 
+  def empty[F[_], K](implicit F: Applicative[F]): KeyDatabase[F, K] =
```

Comment:
The same comment about the style of implicit `Applicative[F]` argument as above.